flask_appbuilder/security/sqla/apis/group/api.py

```diff
@@ -0,0 +1,213 @@
+from flask import request
+from flask_appbuilder import ModelRestApi
+from flask_appbuilder.api import expose, safe
+from flask_appbuilder.const import API_RESULT_RES_KEY
+from flask_appbuilder.models.sqla.interface import SQLAInterface
+from flask_appbuilder.security.decorators import permission_name, protect
+from flask_appbuilder.security.sqla.apis.group.schema import (
+    GroupPostSchema,
+    GroupPutSchema,
+)
+from flask_appbuilder.security.sqla.models import Group, Role, User
+from marshmallow import ValidationError
+from sqlalchemy.exc import IntegrityError
+
+
+class GroupApi(ModelRestApi):
+    resource_name = "security/groups"
+    openapi_spec_tag = "Security Groups"
+    class_permission_name = "Group"
+    datamodel = SQLAInterface(Group)
+    allow_browser_login = True
+
+    list_columns = ["id", "name", "label", "description", "roles", "users"]
+    show_columns = list_columns
+    edit_columns = ["name", "label", "description", "users", "roles"]
+    add_columns = edit_columns
+    search_columns = list_columns
+
+    add_model_schema = GroupPostSchema()
+    edit_model_schema = GroupPutSchema()
+
+    @expose("/", methods=["POST"])
+    @protect()
+    @safe
+    @permission_name("post")
+    def post(self):
+        """Create new group
+        ---
+        post:
+          requestBody:
+            description: Model schema
+            required: true
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/{{self.__class__.__name__}}.post'
```

Comment:
don't think this will use `GroupPostSchema` like this. I know User API is like this but seems like a bug, can you check?